File: src/main/java/coza/trojanc/receipt/template/process/fields/ProcessedLine.java
package coza.trojanc.receipt.template.process.fields;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Charl Thiem
 */
public class ProcessedLine implements ProcessedLineItem{

	private List<ProcessedText> lineItems = new ArrayList<>();

	public List<ProcessedText> getLineItems() {
		return lineItems;
	}
}

File: src/main/java/coza/trojanc/receipt/template/process/fields/ProcessedLine.kt
package coza.trojanc.receipt.template.process.fields

import java.util.ArrayList

/**
 * @author Charl Thiem
 */
class ProcessedLine : ProcessedLineItem {

    private val lineItems: MutableList<ProcessedText> = ArrayList<ProcessedText>()

    fun getLineItems(): List<ProcessedText> {
        return lineItems
    }

    fun addProcessedText(processedText: ProcessedText){
        this.lineItems.add(processedText);
    }
}
